pages/understanding-json-schema/reference/regular_expressions.md

```diff
@@ -47,7 +47,7 @@ described below.
     preceding regular expression.
 -   `{x,}`: Match `x` occurrences or more of the preceding regular
     expression.
--   `{x}?`, `{x,y}?`, `{x,}?`: Lazy versions of the above expressions.
+-   `{x,y}?`, `{x,}?`: Lazy versions of the above expressions.
```

Comment:
```suggestion
-   `{x,y}?`, `{x,}?`: Non-greedy versions of the above expressions.
```

While we're here, I'd like to make this fix (it's the only occurrence in the file).

Reply:
Otherwise, your edit is correct.